File: Bedu/Java/Clase2/Calculadora/app/src/main/java/com/charlye934/calculadoraclase2/MainActivity.java
package com.charlye934.calculadoraclase2;

import androidx.appcompat.app.AppCompatActivity;

import android.os.Bundle;
import android.view.View;
import android.widget.Button;
import android.widget.EditText;
import android.widget.Toast;

public class MainActivity extends AppCompatActivity {

    EditText txtNum1, txtNum2;
    Button btnSuma, btnResta, btnMultiplicar, btnDividir;

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);

        txtNum1 = (EditText)findViewById(R.id.txtNum1);
        txtNum2 = (EditText)findViewById(R.id.txtNum2);

        btnSuma = (Button)findViewById(R.id.btnSumar);
        btnResta =(Button)findViewById(R.id.btnRestar);
        btnMultiplicar = (Button)findViewById(R.id.btnMultiplicar);
        btnDividir = (Button)findViewById(R.id.btnDividir);



        btnSuma.setOnClickListener(
                new View.OnClickListener(){

                    @Override
                    public void onClick(View v) {
                        int num1Int = Integer.parseInt(txtNum1.getText().toString());
                        int num2Int = Integer.parseInt(txtNum2.getText().toString());
                        Toast.makeText(getApplicationContext(),"Suma: " + (num1Int+num2Int),Toast.LENGTH_LONG).show();
                    }
                }
        );

        btnResta.setOnClickListener(
                new View.OnClickListener(){

                    @Override
                    public void onClick(View v) {
                        int num1Int = Integer.parseInt(txtNum1.getText().toString());
                        int num2Int = Integer.parseInt(txtNum2.getText().toString());
                        Toast.makeText(getApplicationContext(),"Resta: " + (num2Int-num2Int),Toast.LENGTH_LONG).show();
                    }
                }
        );

        btnMultiplicar.setOnClickListener(
                new View.OnClickListener(){

                    @Override
                    public void onClick(View v) {
                        int num1Int = Integer.parseInt(txtNum1.getText().toString());
                        int num2Int = Integer.parseInt(txtNum2.getText().toString());
                        Toast.makeText(getApplicationContext(),"Multiplicacion: " + (num2Int*num2Int),Toast.LENGTH_LONG).show();
                    }
                }
        );

        btnDividir.setOnClickListener(
                new View.OnClickListener(){

                    @Override
                    public void onClick(View v) {
                        int num1Int = Integer.parseInt(txtNum1.getText().toString());
                        int num2Int = Integer.parseInt(txtNum2.getText().toString());
                        Toast.makeText(getApplicationContext(),"Division: " + (num2Int/num2Int),Toast.LENGTH_LONG).show();
                    }
                }
        );

    }
}

File: Bedu/kotlin/Clase1-2/Calculadora/app/src/main/java/com/charlye934/calculadorakotlin/MainActivity.kt
package com.charlye934.calculadorakotlin

import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle
import android.util.Log
import android.view.View
import android.widget.Toast
import kotlinx.android.synthetic.main.activity_main.*

class MainActivity : AppCompatActivity() {

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)
    }

    fun operacion(view: View) {
        var num1_String = txtNum1.text.toString()
        var num2_String = txtNum2.text.toString()
        var result:String
        var tag = view.tag

        if(num1_String.isEmpty() || num2_String.isEmpty())
            result = "Favor de agregar un valor en ambos campos"
        else{
            var num1 = num1_String.toDouble()
            var num2 = num2_String.toDouble()
            result = when(tag){
                "suma" -> "suma = " + (num1 + num2)
                "resta" -> "resta = " + (num1 - num2)
                "multi" -> "multiplicacion = " + (num1 * num2)
                "divi" -> "division = " + (num1 / num2)
                else -> ""
            }
        }
        Toast.makeText(this,result,Toast.LENGTH_SHORT).show()
    }
}
